vanilla-framework/vanilla-framework
# 1888 pull quote

## Done

Added large and small variants to the pull-quote pattern and made the CSS more "Vanilla" (not targeting HTML elements, etc)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/pull-quotes/
- Check the three component variants against the [design spec](https://github.com/vanilla-framework/vanilla-framework/issues/1888)

## Details

Fixes #1888
